feat: add flags of the world (2.35) DHIS2-7621

Philip-Larsen-Donnelly opened the pull request:
Flags of the world added programmatically from https://flagpedia.net/

- existing country/flag combinations have not been updated; only new ones added.
- flagpedia country names were slugified to create the keys. In a couple of cases this resulted in a different key for a country that already existed. I left these as alternatives, as I think it is harmless. (e.g. there are now two versions of Trinidad and Tobago)
- the images are PNGs with a fixed width of 160px.

Philip-Larsen-Donnelly commented:
Note: the linked Jira issue was only requesting Madagascar :)

larshelge commented:
Looks great @Philip-Larsen-Donnelly , there is conflict now in `i18n_global.properties` otherwise ready to go.

Philip-Larsen-Donnelly commented:
Ugh! Thanks. The original file had DOS line endings, so my update looked like I changed the entire file rather than just the flags section. I have fixed the conflict and merged the other changes (new relative periods).
